### ISSUE TITLE
Noise input

### ISSUE THREAD

**Issue by ShuvenduBikash** [eriklindernoren/PyTorch-GAN#14]
Original paper used a noise with the input of the generator. why did you not use it?

https://github.com/eriklindernoren/PyTorch-GAN/blob/3a009005f1a45f9f1547b89005933cd6d5ca865c/implementations/pix2pix/pix2pix.py#L127

**Comment by eriklindernoren** [comment]
In the paper they use dropout instead of sampled noise as input to the generator.

> In initial experiments, we did not
> find this strategy effective – the generator simply learned
> to ignore the noise – which is consistent with Mathieu et
> al. [39]. Instead, for our final models, we provide noise
> only in the form of dropout, applied on several layers of our
> generator at both training and test time

If you look at the generator's definition you can see that I have used dropout in several layers of the generator.

**Comment by ShuvenduBikash** [comment]
I got it. Thanks :)
